### ISSUE TITLE
Support for additional math functions: min, max, sum

### ISSUE THREAD

**Issue by jaysson** [brick/math#22]
It would be convenient to have these methods in library.

```
BigDecimal::maxOf([1, 0.1, BigDecimal::of('2'), '3']); // BigDecimal::of(3)
BigDecimal::minOf([1, 0.1, BigDecimal::of('2'), '3']); // BigDecimal::of(0.1)
BigDecimal::sumOf([1, 0.1, BigDecimal::of('2'), '3']); // BigDecimal::of(6.1)
```

**Comment by BenMorel** [comment]
You already have [min()](https://github.com/brick/math/blob/510f506185d4c988f97180426ec80fb3dbf51e04/src/BigNumber.php#L135) and [max()](https://github.com/brick/math/blob/510f506185d4c988f97180426ec80fb3dbf51e04/src/BigNumber.php#L165), inherited from `BigNumber`!

```php
echo BigDecimal::max(1, 0.1, BigDecimal::of('2'), '3'); // BigDecimal::of(3)
echo BigDecimal::min(1, 0.1, BigDecimal::of('2'), '3'); // BigDecimal::of(0.1)
```

I agree it's missing `sum()` though!

**Comment by BenMorel** [comment]
`BigNumber::sum()` implemented in fc1ae88cc8e48e19892fbbc7bdaa8ba79707737a and 44d14bb2a1976b6a165d00eb08823eaa6e8f366e.

**Comment by BenMorel** [comment]
Released as version [0.8.6](https://github.com/brick/math/releases/tag/0.8.6)!

**Comment by jaysson** [comment]
That was quick! Thank you so much.
